### ISSUE TITLE
Add Healthcheck

### ISSUE THREAD

**Issue by Freekers** [qdm12/cod4-docker#11]
Perhaps you might be interested in the Healthcheck I added to my image. It checks if the COD4 server is still up/responding by sending 'getinfo' via RCON (using netcat) to the server every 30 seconds. 
If for some reason the container would still run, but the healthcheck fails (e.g. the CoD4 server process is frozen), the Docker container will be marked as 'unhealthy'. Using a Docker image called 'autoheal' it will automatically restart the container.

Since our repos are too far apart, I cannot do a proper pull request, but feel free to copy the bits you need from my fork :)

**Comment by qdm12** [comment]
Yes that's definitely something I wanted to add next :wink: Thanks!
